### PR TITLE
Nits on phpdoc, use of internal methods with unexpected type

### PR DESCRIPTION
**Pull request by TysonAndre**
detected by phan

### CODE REVIEW

File: library/Requests/Cookie/Jar.php [hunk 1]
```diff
@@ -68,7 +68,7 @@ public function offsetExists($key) {
 	 * Get the value for the item
 	 *
 	 * @param string $key Item key
-	 * @return string Item value
+	 * @return string|null Item value (Not null when offsetExists is true)
```
Comment (rmccue) [inline library/Requests/Cookie/Jar.php]: This reads backwards to me; should be `(null if offsetExists is false)`

File: library/Requests/Utility/CaseInsensitiveDictionary.php [hunk 2]
```diff
@@ -46,7 +46,7 @@ public function offsetExists($key) {
 	 * Get the value for the item
 	 *
 	 * @param string $key Item key
-	 * @return string Item value
+	 * @return string|null Item value (always a string if offsetExists is true)
```
Comment (rmccue) [inline library/Requests/Utility/CaseInsensitiveDictionary.php]: Again here, `(null if offsetExists is false)`

File: library/Requests/IRI.php [hunk 3]
```diff
@@ -67,47 +67,47 @@ class Requests_IRI {
...
-	 * @var string
+	 * @var string|null
 	 */
 	protected $port = null;
 
 	/**
 	 * ipath
 	 *
-	 * @var string
+	 * @var string|null
```
Comment (jrfnl) [inline library/Requests/IRI.php]: Is this correct ? The default for the property is an empty string, not `null`.

File: library/Requests/IRI.php [hunk 4]
```diff
@@ -118,6 +118,7 @@ class Requests_IRI {
 	 *
 	 * Each key is the scheme, each value is an array with each key as the IRI
 	 * part and value as the default value for that part.
+	 * @var array
```
Comment (jrfnl) [inline library/Requests/IRI.php]: Please add an empty comment line above the first tag.

**Comment by jrfnl** [comment]
@TysonAndre I'd like to merge this in. Would you mind rebasing the PR so it becomes mergable ?
If you don't want to, don't know how or don't have the time, please let me know.

**Pull request merged**